Tools/LyTestTools/ly_test_tools/o3de/pipeline_utils.py

```diff
@@ -352,31 +352,44 @@ def get_relative_file_paths(start_dir: str, ignore_list: Optional[List[str]] = N
                 all_files.append(os.path.relpath(full_path, start_dir))
     return all_files
 
+def get_differences_between_lists(first: List[str], second: List[str]) -> bool:
+    """
+        Returns two lists that contain unique entries in lists, missing from the other list.
+    """
+    diff_first = []
```

Comment:
There are more compact/performant ways to write this ie

s = set(first)
diff = [x for x in second if x not in s]

or use of the python XOR operator

A lot depends if order preservation is important and if you really need two way diff lists.

Unclear of size of lists you are comparing and if performance matters

Reply:
Right now running the 15 FBX tests on Linux is finishing in roughly 60 seconds for me, so performance isn't yet a concern. I'll look into those other list diff generation options.